python/requirements.txt

```diff
@@ -0,0 +1,4 @@
+pytest==2.9.1
+setuptools==18.5
```

Comment:
shopify_python needs a more recent version of ci.
`setuptools==34.3`
ex. https://github.com/Shopify/starscream/blob/master/requirements/pip.txt#L2

Sorry, we should have mentioned that earlier.